tests/containers/host_configuration.pm

```diff
@@ -26,6 +26,8 @@ sub run {
     }
     else {
         assert_script_run "dhclient -v";
+        my $interface = script_output qq(ip r | awk '{print \$5}' | head -1);
+        validate_script_output "ip a s $interface", sub { m/((\d{1,3}\.){3}\d{1,3}\/\d{1,2})/ };
```

Comment:
I would put those three lines above this condition so they apply for SUSE too.
But no hard feeling, if you think it would require too many VRs just leave it.

Reply:
make totally sense. thanks